### DISCUSSION TITLE
401 Response

### DISCUSSION THREAD

**Discussion by pdu-lab** [tidal-music/discussions#99] [Q&A]
Hi, I am getting this error even though I am copying my client id/secret and getting an access token. Does this have something to do with not having access to certain levels of data? I am making a very simple call...

<Response [401]>
Error fetching album details: 401, Response: {"status":401,"subStatus":6004,"userMessage":"Client does not have required access tier"}

**Comment by aliaksei-taliuk** [comment]
@pdu-lab Hi! Could you please share the request details and your client ID? Indeed, as you guessed - we do have different access levels depending on your client's granted access tier.

**Reply by pdu-lab** [reply]
Thank you! What do I need to do to get access to simple album copyright information?

**Reply by aliaksei-taliuk** [reply]
Copyright information is supposed to be available to everyone, as well as other album info. In order to be able to help you I need to know the details of the HTTP request you're using (URL, HTTP headers).
